reana_job_controller/kubernetes_job_manager.py

```diff
@@ -254,7 +254,7 @@ def _add_krb5_sidecar_container(self, secrets_volume_mount):
                 'subPath': 'krb5.conf'
             }
         ]
-        keytab_file = os.environ.get('HTCONDORCERN_KEYTAB')
```

Comment:
We may need to amend `reana-workflow-controller` since `HTCONDORCERN_USERNAME` is used there as well:

```console
$ grep -C 1 -r HTCONDORCERN_USERNAME .
./reana_workflow_controller/workflow_run_manager.py-        user = \
./reana_workflow_controller/workflow_run_manager.py:            secrets_store.get_secret_value('HTCONDORCERN_USERNAME') or \
./reana_workflow_controller/workflow_run_manager.py-            WORKFLOW_RUNTIME_USER_NAME
```